### DISCUSSION TITLE
Use foreach on namespaces

### DISCUSSION THREAD

**Discussion by raphm4044** [dotnet/csharplang#6043] [Language Ideas]
Hi! I want a new feature for C# : Use foreach() on namespaces

Exemple of what it can do : 

List<Command> cmdList = new List<Command>();

foreach (Command cmd in MyProject.Commands)
{
    cmdList.Add(cmd);
}

**Comment by CyrusNajmabadi** [comment]
What would this mean?  CAn you flesh out your code some more to make it clearer what each part of that snippet is referring to?
